travis/setup_dependencies_common.sh

```diff
@@ -2,7 +2,7 @@
 
 hash -r
 
-set -e
+set -ex
```

Comment:
I'll remove this again once you give me the OK to handle the PIN_FILE move below like I did.